MbDotNet/Models/Responses/ProxyResponse.cs

```diff
@@ -8,9 +8,13 @@ public class ProxyResponse<T> : ResponseBase where T : ResponseFields, new()
         [JsonProperty("proxy")]
         public T Fields { get; set; }
 
-        public ProxyResponse(T fields)
+        [JsonProperty("_behaviors", NullValueHandling = NullValueHandling.Ignore)]
+        public Behavior Behavior { get; set; }
```

Comment:
Can a proxy response have behaviors? It's unclear to me from the documentation. It seems to imply that behaviors are related to responses in general rather than a specific response type, but I'm not sure it makes sense for them to exist on a proxy response. It already includes other methods of adding waits on proxies based on the actual latency of the proxied request: http://www.mbtest.org/docs/api/proxies#proxy-add-wait-behavior

I would probably remove this for now unless we're sure it should be supported here.